src/editor/operations.ts

```diff
@@ -205,8 +208,13 @@ export function formatRangeAsCode(range: Range): void {
             parts.push(partCreator.newline());
         }
     } else {
-        toggleInlineFormat(range, "`");
-        return;
+        if (hasBacktick) {
+            parts.unshift(partCreator.plain("`".repeat(longestBacktickSequence(range.text)+ 1)));
+            parts.push(partCreator.plain("`".repeat(longestBacktickSequence(range.text)+ 1)));
+        } else {
+            toggleInlineFormat(range, "`");
+            return;
+        }
```

Comment:
This is a great start but doesn't work for toggling. You should probably do something like

`toggleInlineFormat(range, "`".repeat(longestBacktickSequence(range.text)+ 1)`

Currently trying to undo the code by clicking the button a 2nd time just deletes the entire text.

Reply:
@t3chguy hi, how about now, below is attached video of it being tested, i will be waiting for your review :

![ezgif com-gif-maker](https://user-images.githubusercontent.com/38439166/161402837-361256ce-6798-4a54-bed8-1c43abe879ea.gif)
)